lua/smart-splits/config.lua

```diff
@@ -26,13 +26,15 @@ local mux_utils = require('smart-splits.mux.utils')
 ---@field ignored_events string[]
 ---@field multiplexer_integration SmartSplitsMultiplexerType|false
 ---@field disable_multiplexer_nav_when_zoomed boolean
+---@field wezterm_cli_path string|nil
 ---@field kitty_password string|nil
 ---@field setup fun(cfg:table)
 ---@field set_default_multiplexer fun():string|nil
 ---@field log_level 'trace'|'debug'|'info'|'warn'|'error'|'fatal'
 
 ---@type SmartSplitsConfig
 local config = { ---@diagnostic disable-line:missing-fields
+  wezterm_cli_path = 'wezterm',
```

Comment:
I think we can still make it out-of-the-box for WSL by doing this in `setup()`

```lua
if mux_utils.is_WSL() then
  -- on WSL default to .exe unless explicitly set in user config
  new_config.wezterm_cli_path = new_config.wezterm_cli_path or 'wezterm.exe'
end
```

Reply:
We can just document this behavior in the code comment in the README.md

Reply:
Sure, that should be okay